src/olympia/devhub/templates/devhub/addons/listing/item_actions.html

```diff
@@ -30,7 +30,7 @@
          title="{{ _('Upload a new version of this add-on.') }}">
         {{ _('New Version') }}</a>
     </li>
-    {% if addon.accepts_compatible_apps() and addon.current_version %}
+    {% if addon.is_listed and addon.accepts_compatible_apps() and addon.current_version %}
```

Comment:
I think this should be testing for latest version... depending on what the compat testing does - at the moment you get compat alerts for both listed and unlisted and _only_ showing when there is a current_version is going to change that when we're only setting current_version for public, listed versions.

This has reminded me we have no plan about how to change the 'my submissions' list to handle a latest version for multiple channels at once.